src/exit_node.cpp

```diff
@@ -18,18 +18,30 @@
 #include <string>
 #include <utility>
 
-#include <spdlog/spdlog.h>
+#include "logging.hpp"
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wall"
 #include "tensorflow/core/framework/tensor.h"
 #pragma GCC diagnostic pop
 
+#include "exitnodesession.hpp"
+
 namespace ovms {
+Status ExitNode::fetchResults(NodeSession& nodeSession, SessionResults& nodeSessionOutputs) {
+    auto& exitNodeSession = static_cast<ExitNodeSession&>(nodeSession);
```

Comment:
Why is it static cast but in dl node there is dynamic_cast?

Reply:
Good that you noticed - > I should have been replace all those with static_cast.